central/debug/service/service.go

```diff
@@ -214,6 +234,31 @@ func (s *serviceImpl) StreamAuthzTraces(_ *v1.Empty, stream v1.DebugService_Stre
 	}
 }
 
+func getJSONToZipWithLogging(ctx context.Context, zipWriter *zip.Writer, fileName string, fetchData func(ctx context.Context) (interface{}, error)) {
```

Comment:
```suggestion
func getJSONToZipWithLogging(ctx context.Context, zipWriter *zip.Writer, fileName string, fetchDataF func(ctx context.Context) (interface{}, error)) {
```
I follow this convention to distinguish between global functions and functor objects / closures. Up to you whether to adopt it as well : ).